pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -327,6 +379,29 @@ func (h *upgradeHandler) cleanup(upgrade *harvesterv1.Upgrade, cleanJobs bool) e
 			return err
 		}
 	}
+
+	// tear down logging infra if any
+	if upgrade.Spec.LogEnabled {
+		upgradeLogName := fmt.Sprintf("%s-upgradelog", upgrade.Name)
```

Comment:
It's better to use a util function to construct `upgradeLogName`.

Reply:
Updated, thanks!